lang/en/ratingallocate.php

```diff
@@ -201,6 +201,9 @@
 $string['choice_maxsize_display'] = 'Maximum number of students';
 $string['choice_title'] = 'Title';
 $string['choice_title_help'] = 'Title of the choice. *Attention* all active choices will be displayed while ordered by title.';
+$string['choice_usegroups'] = 'Restrict visibility by groups';
+$string['choice_usegroups_help'] = 'If selected, this group will only be visible to the members of the specified groups.';
+$string['choice_groupselect'] = 'Select visibility groups';
```

Comment:
let's simplify the choice_groupselect string to "Groups" - makes it similar to other group selectors in core.

Reply:
Sure. Updated in commit 14d5729.